Exporter/src/main/resources/application.yml

```diff
@@ -158,7 +159,7 @@ jobs:
     queryModified:
     notificationUrl: http://ves-ebi-6f:8080/biostudies/api/v1/index/reload/default
     cron: 0 0 19 * * ? # everyday at 7 p.m.
-    recordsThreshold: 500000
+    recordsThreshold: 50000
```

Comment:
Are you sure about this? I think we should leave it at 500K since we have over 1M submissions